### PR TITLE
Uml 867 code cancelled confirmation flash message

### PR DESCRIPTION
**Pull request by GemTay**
# Purpose

Displays a flash message on the check access codes page to provide a confirmation to users once they have cancelled a code.

Fixes UML-867

## Approach

Constructed a flash message in the CancelCodeHandler that is then captured by the CheckAccessCodesHandler and rendered. Behat tests extended to check that the flash message appears when a code is cancelled.

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [x] I have added tests to prove my work
* [x] I have added welsh translation tags and updated translation files
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/app/src/Actor/src/Handler/CancelCodeHandler.php [hunk 1]
```diff
@@ -75,9 +86,32 @@ public function handle(ServerRequestInterface $request): ResponseInterface
...
 
-            return new RedirectResponse($this->urlHelper->generate('lpa.access-codes',[],['lpa' => $validated['lpa_token']]));
+            /** @var FlashMessagesInterface $flash */
+            $flash = $request->getAttribute(FlashMessageMiddleware::FLASH_ATTRIBUTE);
+
+            $formattedCode = str_split($validated['viewer_code'], 4);
+            array_unshift($formattedCode, 'V');
+            $formattedCode = implode("-", $formattedCode);
+
+            $message = $this->translator->translate(
```
Comment (cooperaj) [inline service-front/app/src/Actor/src/Handler/CancelCodeHandler.php:96]:
```suggestion
            $message = $this->translator->translate(
                "You cancelled the access code for %organisation%: %code%",
                [
                    '%code%' => $formattedCode,
                    '%organisation%' => $validated['organisation']
                ],
                null,
                'flashMessage'
            );
```

File: service-front/app/src/Actor/src/Handler/CancelCodeHandler.php [hunk 2]
```diff
@@ -75,9 +86,32 @@ public function handle(ServerRequestInterface $request): ResponseInterface
...
+                "You cancelled the access code for %organisation%: %code%",
+                [
+                    '%code%' => $formattedCode,
+                    '%organisation%' => $validated['organisation']
+                ]
+            );
```
Comment (cooperaj) [inline service-front/app/src/Actor/src/Handler/CancelCodeHandler.php:102]:
```suggestion
            $message = $this->translator->translate(
                "You cancelled the access code for %organisation%: %code%",
                [
                    '%code%' => $formattedCode,
                    '%organisation%' => $validated['organisation']
                ],
                null,
                'flashMessage'
            );
```

Comment (cooperaj) [reply service-front/app/src/Actor/src/Handler/CancelCodeHandler.php:102]: This will fix the flash message issue.

**Review by SeemaMenon — COMMENTED** [review]
LGTM.

Noticed two things.

1. The  button alignment on the confirm cancelation page is not aligned in one line [welsh page] .This is due to length of Welsh text. Mentioned in case we need to bring to notice to Sarah.
2. Flash message does not translate.

**Review by SeemaMenon — APPROVED** [review]

**Pull request merged**